### PR TITLE
Zqs 1171/seprate installations

### PR DESCRIPTION
**Pull request by Boniface316**
## Description

Include description of feature this PR introduces or a bug that it fixes. Include the following information:

Simulators are installed separately from the core. This ensures that the user will not have issues with the simulators they do not need.

## Please verify that you have completed the following steps

- [X] I have self-reviewed my code.
- [X] I have updated documentation.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -38,6 +37,17 @@ dev =
     orquestra-opt[cma]
     orquestra-cirq[qsim]
 
+qulacs =
```
Comment (mstechly) [inline setup.cfg:40]: You should also add the new installations to `dev`.
I'm not sure what the syntax should be, but I think it was something like `.[qiskit]`

File: README.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ To install the latest versions of all of the Orquestra Core packages together ru
 
 `pip install orquestra-core`
 
+Libraries to interact with quantum simulators or QPU needs to be installed seperately. i.e `pip install orquestra[cirq]`.
```
Comment (mstechly) [inline README.md:41]: I'd rephrase it to sth like:
"Libraries to interact with other frameworks (e.g. qiskit, cirq), simulators or QPU can be installed as optional dependencies, i.e.: `pip install orquestra[cirq]`"

**Review by mstechly — CHANGES_REQUESTED** [review]

File: setup.cfg (hunk 1 above)
Comment (Boniface316) [reply setup.cfg:43]: Done

**Review by mstechly — APPROVED** [review]
